Makefile

```diff
@@ -79,7 +79,7 @@ scm-source.json: .git
 
 tools:
 	@go get -u honnef.co/go/tools/cmd/staticcheck
-	@go get -u github.com/Masterminds/glide
+	@go get k8s.io/client-go@kubernetes-1.15.0
```

Comment:
valid point. I'll update it. Btw, I introduced it here, because I'm getting the following error when only relying on `go mod vendor`:
```# k8s.io/client-go/rest
../../../../pkg/mod/k8s.io/client-go@v11.0.0+incompatible/rest/request.go:598:31: not enough arguments in call to watch.NewStreamWatcher
	have (*versioned.Decoder)
	want (watch.Decoder, watch.Reporter)
```

Was also reported [here](https://github.com/kubernetes/client-go/issues/584).